Azure/sonic-buildimage
# [swss.sh] clear counters cache folder on swss cold/fast reload

A change in sonic-utilities makes all cache files be saved into a
/tmp/cache. On swss restart this cache has to be removed in case swss
starts in cold or fast mode. A related cache restoration in the warmboot
finalizer script is also updated to use new location.

Signed-off-by: Stepan Blyschak <stepanb@nvidia.com>



*DEPENDS ON*: https://github.com/Azure/sonic-utilities/pull/2232/

#### Why I did it

To fix https://github.com/Azure/sonic-buildimage/issues/9817. Clear the cache directory on swss.sh except for warm start.
Also, adopted finalize-warmboot script to take the cache directory.

#### How I did it

As described in previous section.

#### How to verify it

Run togather with https://github.com/Azure/sonic-utilities/pull/2232/. Verify counters cache is removed on config reload, cold/fast reboots, swss restart.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
